tox.ini

```diff
@@ -3,36 +3,13 @@ envlist =
     py{36,37,38}
     build_docs
     codestyle
-    conda
+    py38-conda
 isolated_build = true
-requires =
-    setuptools >= 30.3.0
-    pip >= 19.3.1
-    tox-pypi-filter >= 0.10
 
 [testenv]
-# The following option combined with the use of the tox-pypi-filter above allows
-# project-wide pinning of dependencies, e.g. if new versions of pytest do not
-# work correctly with pytest-astropy plugins. Most of the time the pinnings file
-# should be empty.
-pypi_filter_requirements = https://raw.githubusercontent.com/sunpy/package-template/master/.test_package_pins.txt
-# Pass through the following environemnt variables which may be needed for the CI
-passenv = HOME WINDIR LC_ALL LC_CTYPE CC CI TRAVIS
```

Comment:
we need to add CIRCLECI to this on the template.

Again with the chaos causing when we retemplate though.

Reply:
I can revert these changes.

Reply:
Fixed.